tests/scalability/README.md

```diff
@@ -94,3 +94,45 @@ From here we get the ddocs update sequences. Stage the branch which is associate
 Once the medic script completes we begin setting up for jmeter. The `start_jmeter_ec2.sh` script launches a new ec2 instance with the `user-data` script at `cht-core/tests/scalability/run_suite.sh`. 
 
 The script clones cht-core and navigates to the scalability dir. Then installs java, nodejs, jmeter, and its plugins. Executes the scalability suite which is defined in the `cht-core/tests/scalability/sync.jmx`, and finally it uploads the results to an s3 bucket. 
+
+
+
+## Distributed Workflow
+
+![Diagram of our current setup.](jmeter_distributed.png)
+
+Our current setup has 1 ec2 instance acting as the JMeter Orchestrator with 2 worker nodes. This can generate 1400 virtual users. Against our Application environment under test. 
+
+### Setup
+
+1. Launch an ec2 instance c5.2xlarge(Orchestrator)
+1. Launch 2 ec2 instances of c5.4xlarge(Worker)
+1. SSH into each of these machines. 
+1. Copy `tests/scalability/run_suite.sh` to the user directory(`/home/ubuntu`). From this branch.
+1. On each of the three VMs, call  `chmod +x run_suite.sh&&./run_suite.sh` . This clones cht-core, installs JAVA, installs Node 14, runs npm install, installs JMeter, and its plugins. The resulting directory will be `/home/ubuntu/cht-core/tests/scalability`.
+1. On just the orchestrator node, update the `cht-core/tests/scalability/config.json` to ensure the users and URL are correct.
+1. Split the `config.json` on the orchestrator
+    1. `cd cht-core/tests/scalability`
+    1. `npm install` 
```

Comment:
on my container `npm` wasn't installed.  Tracking it back, I saw that the bash script from nodesource.com we're running is failing when it tries to install node:

``` 
ubuntu@deleteme:~/cht-core/tests/scalability$ curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -

======     
   DEPRECATION WARNING    Node.js 10.x is no longer actively supported! 
  [SNIPPED OUT CONTENT HERE}
=======

Continuing in 20 seconds ...

## Installing the NodeSource Node.js 10.x repo...

## Populating apt-get cache...

+ apt-get update
Ign:1 https://deb.nodesource.com/node_10.x focal InRelease
Err:2 https://deb.nodesource.com/node_10.x focal Release                                
  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 23.218.80.177 443]
Hit:3 http://security.ubuntu.com/ubuntu focal-security InRelease                                        
Hit:4 http://archive.ubuntu.com/ubuntu focal InRelease                                                  
Hit:5 http://archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:6 http://archive.ubuntu.com/ubuntu focal-backports InRelease
Reading package lists... Done
E: The repository 'https://deb.nodesource.com/node_10.x focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Error executing command, exiting
```

Given this kinda breaks the rest of stuffs - i'll stop here her for now!

Reply:
Yeah! I think that's part of the changes to the current script and the one in my branch. Sadly they're different enough  but I'm not sure it's great to house two similar scripts. 

Thoughts? 

Here's what the updated one looks like in my other branch that I'm picking changes on. 

https://github.com/medic/cht-core/blob/78969702ca23a88a928d34890cdad269d99f1c66/tests/scalability/run_suite.sh

Reply:
I'd sayd, let's just be verbose and include both script side by side.  We can give them different names and add an item or two to the readme.md to explain how to use them.  If you want, you could normalize them by including shared sections from a single script, and then have the endemic parts local that arent' shared.

Otherwise, running [this new script](https://github.com/medic/cht-core/blob/78969702ca23a88a928d34890cdad269d99f1c66/tests/scalability/run_suite.sh) in a clean container executes perfectly (if not a bit slow downloading jmeter :stuck_out_tongue_winking_eye: )